common/sky/constants.py

```diff
@@ -0,0 +1,32 @@
+from typing import Final
+import astropy.units as u
+from astropy.time import Time
+
+"""
+Constants that are used here and there.  Some are Quantities,
+others are just floats. Not all are used.
+
+The planet-coefs are for series expansions for the phase functions
+of the planets, used in predicting apparent magnitude.  See code.
+"""
+
+PI: Final = 3.14159265358979
```

Comment:
Nice! I'm going to have to back through some of my old code and add this, too.
Thanks for doing it.